.travis.yml

```diff
@@ -5,18 +5,18 @@ python:
   - 3.4
   - 3.5
   - 3.6
+  - 3.7
+  - 3.8
+  - 3.8-dev
+  - 3.9-dev
   - pypy
   - pypy3.5
 matrix:
-  include:
-  # See https://github.com/travis-ci/travis-ci/issues/9069
-  # for more information.
-  - python: 3.7
-    sudo: required
-    dist: xenial
-  - python: 3.8-dev
-    sudo: required
-    dist: xenial
+  fast_finish: true
+  allow_failures:
+    - python: 3.8
```

Comment:
> Once 3.8.0 is out as a stable version we'll stop allow failures on it.

so we can remove this, right?

Reply:
Unfortunately not yet, Python 3.8.0 is not yet available on Travis

Reply:
Let's wait https://travis-ci.community/t/add-python-3-8-support/5463

Reply:
+1

Reply:
we're ready to go https://travis-ci.community/t/add-python-3-8-support/5463/6